packages/ember-toucan-core/src/-private/components/form/controls/combobox/option.hbs

```diff
@@ -0,0 +1,36 @@
+{{! template-lint-disable require-presentational-children }}
+<li
+  aria-selected={{if @isSelected "true" "false"}}
+  class="my-0 flex cursor-default items-center gap-2 px-2 py-2 leading-4
+    {{this.styles}}
+    {{this.className}}
+    "
+  data-active={{if @isActive "true" "false"}}
+  id="{{@popoverId}}-{{@index}}"
+  role="option"
+  {{on "click" this.onClick}}
+  {{! template-lint-disable no-pointer-down-event-binding }}
+  {{on "mousedown" this.onMousedown}}
+  {{on "mouseover" @onMouseover}}
+  ...attributes
+>
+  <this.Check aria-hidden="true" class={{unless @isSelected "invisible"}} />
+  {{yield}}
+
+  {{! TODO: Do we make `@value` required? }}
+  {{!
+      We'll need Form::Controls::Select to work with real forms and the native `FormData` API.
+      That means it'll need to be backed by a real INPUT. Screenreaders already have all the information
+      they need, which means the INPUT is superfluous and is thus noise?
+  }}
+  {{! template-lint-disable no-nested-interactive }}
+  {{! template-lint-disable require-input-label }}
+  <input
+    class="hidden"
+    checked={{@isSelected}}
+    disabled={{@isDisabled}}
+    readonly={{@isReadOnly}}
+    type="checkbox"
+    value={{@value}}
+  />
```

Comment:
I'm a bit unclear on how we feel about this as a group.  Clinton added this super helpful comment about FormData.  I don't have any strong opinions leaving this in here or ripping it out.  What do y'all think?

Reply:
I like the super helpful comment. I'd prefer `<input>` or `<input />` over `INPUT` because I was thinking that `INPUT` refers to a special `INPUT` component we've created (got distracted with how it's written). Also maybe something about how a `hidden` input is required to satisfy the FormData API (users don't need to see the input, the FormData api does)

Maybe that's just me tho...I had to read this a few times to connect all the parts.